src/throttler.guard.ts

```diff
@@ -17,7 +17,7 @@ export class ThrottlerGuard implements CanActivate {
   constructor(
     @InjectThrottlerOptions() protected readonly options: ThrottlerModuleOptions,
     @InjectThrottlerStorage() protected readonly storageService: ThrottlerStorage,
-    protected readonly reflector: Reflector,
+    @Inject(Reflector.name) protected readonly reflector: Reflector,
```

Comment:
https://github.com/nestjs/nest/issues/7583#issuecomment-883166541

Reply:
I feel that one could have registered a provider with the token `"Reflector"` and then he/she will get a bunch of errors on `ThrottlerGuard`

I didn't follow why the class ref aproach isn't the right one :thinking: